src/features/surveys/components/SurveyForm/OptionsQuestion.tsx

```diff
@@ -142,9 +142,13 @@ const OptionsQuestion: FC<OptionsQuestionProps> = ({
                 value={dropdownValue}
               >
                 {options.map((option: ZetkinSurveyOption) => (
-                  <MenuItem key={option.id} value={option.id}>
+                  <MenuItem
+                    key={option.id}
+                    sx={{ whiteSpace: 'normal' }}
+                    value={option.id}
+                  >
                     <Typography
-                      noWrap
+                      flex="wrap"
```

Comment:
I think this solution is great! I wasn't aware it was possible to wrap in a drop down, but of course that makes sense for MUI, so lets do this. Are the other changes in this PR needed?